synapse/storage/data_stores/main/schema/delta/59/pushrules_enabled_delete_obsolete.sql

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2020 The Matrix.org Foundation C.I.C.
```

Comment:
I think this file is in the wrong directory; should be /58/ ?

also can it have a number please.